### PR TITLE
New UI added.

### PR DESCRIPTION
**Pull request by Abhirup-99**
This PR should be merged after the UI updates PR, as it is built upon it.

### CODE REVIEW

File: src/FormHolder.tsx [hunk 1]
```diff
@@ -14,14 +14,14 @@ const FormHolder = ({
 }) => (
   <div className="iconDialog">
     <h1 className="h1">
-      <span>EOS</span>
+      <img
+        src="https://eos-icons.com/static/media/eos-icons-logo.b8cf23f8.svg"
```
Comment (kbhutani0001) [inline src/FormHolder.tsx:18]: please download and use the file. This url could change on next build.

File: src/assets/info-circle-solid.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="info-circle" class="svg-inline--fa fa-info-circle fa-w-16" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path fill="currentColor" d="M256 8C119.043 8 8 119.083 8 256c0 136.997 111.043 248 248 248s248-111.003 248-248C504 119.083 392.957 8 256 8zm0 110c23.196 0 42 18.804 42 42s-18.804 42-42 42-42-18.804-42-42 18.804-42 42-42zm56 254c0 6.627-5.373 12-12 12h-88c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h12v-64h-12c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h64c6.627 0 12 5.373 12 12v100h12c6.627 0 12 5.373 12 12v24z"></path></svg>
```
Comment (kbhutani0001) [inline src/assets/info-circle-solid.svg:1]: if this is the `info` icon, please use it from eos-icons-react

Comment (kbhutani0001) [reply src/assets/info-circle-solid.svg:1]: https://storybook.eos-icons.com/?path=/story/icon-eos-info--default&args=size:xxxl

**Comment by Abhirup-99** [comment]
@kbhutani0001 I have done the required changes. Do take a look when free. Thanks :)

**Review by kbhutani0001 — APPROVED** [review]

**Pull request merged**